### DISCUSSION TITLE
Does the package support to odoo14?

### DISCUSSION THREAD

**Discussion by akira32** [foxglove/community#1031] [General]
Does the package support to odoo14?

**Comment by defunctzombie** [comment]
Please elaborate on your question. What is Odoo14? What do you mean by "support"? Which package? We have many packages we release.

**Reply by jtbandes** [reply]
Based on [this previous question](https://github.com/orgs/foxglove/discussions/1029) I am assuming this question refers to `@foxglove/xmlrpc`.

The xmlrpc package does not have any special relation to odoo, it is just meant to provide simple standard xmlrpc support. If you are running into some issue, you'll need to provide more details about what you believe the package is doing wrong.

**Reply by akira32** [reply]
Odoo is HRP Web can use xmlrpc to access data.
